doc/tutorials/active_dendrite/nestml_active_dendrite_tutorial.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NESTML active dendrite tutorial\n",
+    "\n",
+    "In this tutorial, we create a neuron model with a \"nonlinear\" or \"active\" dendritic compartment, that can, independently from the soma, generate a dendritic action potential. Instead of modeling the membrane potential of the dendritic compartment explicitly, the dendritic action potential (dAP) is modeled here as the injection of a rectangular (pulse shaped) dendritic current into the soma, parameterized by an amplitude and a duration. The rectangular shape can be interpreted as the approximation of an NMDA spike (Antic et al. 2010). A dendritic action potential is triggered when the total synaptic current exceeds a threshold.\n",
+    "\n",
+    "The model we are building is an adapted version of the neuron model introduced by Memmesheimer et al. (2012) and Jahnke et al. (2012).\n",
+    "\n",
+    "**Table of contents**\n",
+    "- [Pulse-shaped dendritic action potential](#section_pulse)\n",
+    "- [Dynamically controlling synaptic integration](#section_integration)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib as mpl\n",
+    "import matplotlib.pyplot as plt\n",
+    "import nest\n",
+    "import numpy as np\n",
+    "import os\n",
+    "from pynestml.frontend.pynestml_frontend import to_nest, install_nest\n",
+    "\n",
+    "NEST_SIMULATOR_INSTALL_LOCATION = nest.ll_api.sli_func(\"statusdict/prefix ::\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='section_pulse'></a>\n",
+    "## Pulse-shaped dendritic action potential\n",
```

Comment:
the dAP **current** is pulse-shaped and not the dAP itself. So, instead of writing "dendritic action potential", you could write "dendritic action potential currents" or "dAP currents".